pkg/iostreams/color.go

```diff
@@ -121,3 +121,32 @@ func (c *ColorScheme) SuccessIcon() string {
 func (c *ColorScheme) WarningIcon() string {
 	return c.Yellow("!")
 }
+
+func (c *ColorScheme) ColorFromString(s string) func(string) string {
```

Comment:
i may not keep this around, was just trying to avoid passing a ColorScheme to as many small helper functions